lomond/websocket.py

```diff
@@ -89,6 +89,13 @@ def session(self):
     def key(self):
         return self.state.key
 
+    def __enter__(self):
```

Comment:
Allows the use of websocket as a context manager, which will close the socket if an exception is unhandled. Otherwise it will only close when the websocket is garbage collected.